lib/arraySorting.js

```diff
@@ -1,11 +1,14 @@
-const names = ["Alice", "Bob", "Jeremy", "Sam", "Henry", "Sarah", "Ashely"];
+const name = ["Alice"];
 // const names = [];
 
 function Printname() {
```

Comment:
You most likely want an argument here so you can pass the name you are wanting to test with.
`function Printname(name)`